sda-commons-server-openapi/src/test/java/org/sdase/commons/server/openapi/OpenApiBundleDefaultPortIT.java

```diff
@@ -9,21 +9,21 @@
 
 import io.dropwizard.Configuration;
 import io.dropwizard.testing.ConfigOverride;
-import io.dropwizard.testing.junit.DropwizardAppRule;
+import io.dropwizard.testing.junit5.DropwizardAppExtension;
 import javax.ws.rs.client.Invocation.Builder;
 import javax.ws.rs.core.Response;
-import org.junit.ClassRule;
-import org.junit.Ignore;
-import org.junit.Test;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
 import org.sdase.commons.server.openapi.apps.test.OpenApiBundleTestApp;
 import org.sdase.commons.server.openapi.test.OpenApiAssertions;
 
-@Ignore(
+@Disabled(
     "This test can not be run on the build server since it binds to port 80 which is not permitted (Permission denied)")
-public class OpenApiBundleDefaultPortIT {
-  @ClassRule
-  public static final DropwizardAppRule<Configuration> DW =
-      new DropwizardAppRule<>(
+class OpenApiBundleDefaultPortIT {
```

Comment:
Strange test. I guess it was not executed for a couple of years. @JoergSiebahn Can you remember the purpose of the test?

Reply:
Maybe it's about to verify that the port is [not part of the server list](https://github.com/SDA-SE/sda-dropwizard-commons/pull/1952/files#diff-0769157ed7fcff783f6a77e2b0ebf927def64e94247d29548b0e0c08d7482820R58)?